c4_build_catalog.m

```diff
@@ -0,0 +1,44 @@
+
+cooksey_c4_catalog = fitsread('/home/reza/gp/c4/Cooksey_c4_cat/c4_catalogs/distfiles/Cooksey_C4_detected.fits', 'binarytable');
```

Comment:
Could you write up a downloading script for this file `Cooksey_C4_detected.fits` and mkdir this directory?
Or at least avoiding hard-code the path.

The path of the directory could be `data/dr7q/civ_catalogs/` something like that.

Reply:
Actually, this fits file is not available for download anywhere. I made it myself by downloading the table in Cooksey's paper and converting that to fits file. What about uploading that somewhere and then, as you said, writing up a script for that? I am not sure where I should upload that...

I fixed the path issue.

Reply:
If the file is not too large maybe a separate repo would be fine, like sbird did in here https://github.com/sbird/dla_data

If it is large, then writing up a downloading script from the source would be good.

If it is not easy to download then we can put in a Google drive or dropbox, just make sure people can find it.

Reply:
For spectra which ~4-5GB when can write a script for download. but for this two fits catalogs which are not big (totoally ~4MB) I can put them in the repo directly.